### DISCUSSION TITLE
MariaDB support

### DISCUSSION THREAD

**Discussion by cweinreben** [slick/slick#2882] [Questions]
When Debian switched from MySQL to MariaDB with version 9, we also changed our database to MariaDB. Is there a reason why MariaDB is not mentioned anywhere in the Slick documentation? Can we just use the MariaDB Connector/J with Slick?

**Comment by nafg** [comment]
If it's fully compatible with MySQL then using the MySQL profile should work fine.

In any case contributions are welcome

**Reply by cweinreben** [reply]
We can test using the current MySQL driver. An extra support for MariaDB is too much effort I guess. I checked the MariaDB page. https://mariadb.com/kb/en/mariadb-vs-mysql-compatibility/ They mention "MariaDB's client protocol is binary compatible with MySQL's client protocol". I guess it should work. I will try to run the tests against a MariaDB database.

**Reply by nafg** [reply]
Yeah Maria is a fork of MySQL IIRC

Not sure what you mean by "too much effort." For who?

**Comment by qomosoloto** [comment]
Is there any toturial about how to support a new kind database?
